pkgs/servers/sql/mariadb/default.nix

```diff
@@ -50,39 +50,37 @@ common = rec { # attributes common to both builds
   cmakeFlags = [
     "-DBUILD_CONFIG=mysql_release"
     "-DMANUFACTURER=NixOS.org"
-    "-DDEFAULT_CHARSET=utf8"
-    "-DDEFAULT_COLLATION=utf8_general_ci"
+    "-DDEFAULT_CHARSET=utf8mb4"
+    "-DDEFAULT_COLLATION=utf8mb4_unicode_ci"
     "-DSECURITY_HARDENED=ON"
 
-    "-DMYSQL_UNIX_ADDR=/run/mysqld/mysqld.sock"
+    "-DINSTALL_UNIX_ADDRDIR=/run/mysqld/mysqld.sock"
```

Comment:
Is this correct? `mysqld.sock` is a socket file, not a directory (as suggested by the name `ADDRDIR`).

Reply:
Yes. If not specified "MYSQL_UNIX_ADDR" is used INSTALL_UNIX_ADDRDIR
https://github.com/MariaDB/server/blob/10.3/cmake/install_layout.cmake#L252-L254